source/models/users/User.js

```diff
@@ -1,7 +1,7 @@
 const { DataTypes: { STRING, BOOLEAN }, Model } = require('sequelize');
 
 /** This class stores global information for bot users */
-let userModel = {
+const userModel = {
```

Comment:
Any particular reason for this switch?

Reply:
It would suck if `userModel` got reassigned accidentally, and this would get the terminal to comment if it does.

Reply:
Yep, makes sense.